assets/css/lesson.scss

```diff
@@ -145,6 +145,14 @@ img {
 // Miscellaneous.
 //----------------------------------------
 
+article.figures p {
+  text-align: center;
+}
+article.figures img {
+  display: block;
+  margin: 20px auto;
+}
```

Comment:
I think we also need a `max-width: 100%`

Reply:
There is no need for this block in its entirety, actually, because of 5e4ca8b.

Reply:
:+1: